src/Components/Task.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC, useState } from 'react'
+import {
+    Box,
+    Checkbox,
+    ListItem,
+    Typography
+} from '@material-ui/core'
+
+import DeleteIcon from '@material-ui/icons/Delete';
+
+interface Props {
+    task: {
+        title: string,
+        taskId: number
```

Comment:
I suggest changing to `id` to avoid extra context.
```suggestion
        id: number
```

Reply:
Resolved on 6d2412f5a502b689afba8b14f36edacbb9ccf8fb